src/components/footnote-link/footnote-link.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import template from './demo/demo.twig';
+// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
+// See: https://github.com/webpack-contrib/raw-loader#examples
+// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
+// okay with the following Webpack-specific raw loader syntax. It's better to leave
+// the ESLint rule enabled globally, and only thoughtfully disable as needed (e.g.
+// within a Storybook docs page and not within an actual component).
+// This can be revisited in the future if Storybook no longer relies on Webpack.
+// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
+import footnoteLinkSource from '!!raw-loader!./demo/demo.twig';
+
+<Meta
+  title="Components/Footnote/Footnote Link"
+  argTypes={{
+    count: { type: { name: 'number' } },
+    id: { type: { name: 'string' } },
+  }}
+/>
+
+# Footnote Link
+
+A link to a footnote. Intended to be matched with a [Footnote](/docs/components-footnote-footnote--basic).
+
+## Basic usage
+
+<Canvas>
+  <Story
+    name="Basic"
+    args={{ count: 1 }}
+    parameters={{
+      docs: {
+        source: {
+          code: footnoteLinkSource,
+        },
+      },
+    }}
+  >
+    {(args) => template(args)}
+  </Story>
+</Canvas>
+
+## Template Properties
+
+- `count` (number)
```

Comment:
@Paul-Hebert Could you add a description to `count`? 🙂

Reply:
Yep!

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/c2a2cd9738e947e7f6930d4b5650777274ee6b11